### PR TITLE
Fix dashboard so that data is displayed correctly

### PR DESCRIPTION
**Pull request by DaliHoffman**

### CODE REVIEW

File: js/dashboard.js [hunk 1]
```diff
@@ -61,12 +61,12 @@ fetch('https://api.weatherbit.io/v2.0/forecast/daily?lat=-45.874&lon=170.503&key
 .then(response =>         
     response["data"].filter(day => day.datetime == formatDate(testSaturday(new Date())) || day.datetime == formatDate(testSunday(new Date())) || day.datetime == getToday())
     .forEach((data, i) => {
-
     //date that needs to be formatted to say the day
-
+        console.log('data', data)
```
Comment (Jordann-Hand) [inline js/dashboard.js:65]: Is this needed?

File: js/dashboard.js [hunk 2]
```diff
@@ -61,12 +61,12 @@ fetch('https://api.weatherbit.io/v2.0/forecast/daily?lat=-45.874&lon=170.503&key
...
     let datetime = document.createElement("div");
     datetime.innerHTML = reformatDate(data.datetime);
     arrowGrid.appendChild(datetime);
+    console.log('data.datetime', data.datetime)
```
Comment (Jordann-Hand) [inline js/dashboard.js:69]: Same here

File: js/dashboard.js [hunk 3]
```diff
@@ -171,21 +172,23 @@ fetch(buildNIWA_URL(NIWA_PATHS.data, currentLocation, 31)).then(response => resp
         makeDay("div", tchange, "green")        
 
         //saturday div
+        console.log('uniquedays', uniquedays)
```
Comment (Jordann-Hand) [inline js/dashboard.js:175]: Same here

File: js/dashboard.js [hunk 4]
```diff
@@ -199,6 +202,8 @@ fetch(buildNIWA_URL(NIWA_PATHS.data, currentLocation, 31)).then(response => resp
 
         let sundaychange = (sumax - sumin).toFixed(2) + "m"
         makeDay("div", sundaychange, "green")
+        console.log('smin, smax, satchange', smin, smax, satchange)
```
Comment (Jordann-Hand) [inline js/dashboard.js:205]: same here

File: js/dashboard.js [hunk 5]
```diff
@@ -199,6 +202,8 @@ fetch(buildNIWA_URL(NIWA_PATHS.data, currentLocation, 31)).then(response => resp
...
+        console.log('smin, smax, sdunday', sumin, sumax, sundaychange)
```
Comment (Jordann-Hand) [inline js/dashboard.js:206]: and again

**Review by Jordann-Hand — CHANGES_REQUESTED** [review]
Code:
Code looks good, a couple console logs are they needed?
Function:
Works nicely, nice fix up 
Style: 
Nice style, looks better than what it originally did
Summary: 
Looks awesome. Good stuff. Will wait to here back about the console logs then will approve.

File: js/dashboard.js (hunk 1 above)
Comment (DaliHoffman) [reply js/dashboard.js]: All done man @Jordann-Hand  Got them all

**Review by Jordann-Hand — APPROVED** [review]

**Pull request merged**